src/main/java/org/candlepin/resteasy/filter/AuthenticationFilter.java

```diff
@@ -164,10 +168,12 @@ public void filter(ContainerRequestContext requestContext)
                 principal = new NoAuthPrincipal();
             }
             else if (!config.getBoolean(ConfigProperties.AUTH_OVER_HTTP) && !request.isSecure()) {
-                throw new BadRequestException("Please use SSL when accessing protected resources");
+                throw new BadRequestException(
+                    i18n.tr("Please use SSL when accessing protected resources"));
```

Comment:
This doesn't have to be in a new line; there's enough space on the previous one I believe

Reply:
Fixed